compiler/aarch64/codegen/ARM64SystemLinkage.cpp

```diff
@@ -627,7 +627,7 @@ TR::ARM64SystemLinkage::createEpilogue(TR::Instruction *cursor)
       }
 
    // return
-   cursor = generateRegBranchInstruction(codeGen, TR::InstOpCode::ret, lastNode, lr, cursor);
+   cursor = generateRegBranchInstruction(codeGen, TR::InstOpCode::retn, lastNode, lr, cursor);
```

Comment:
This line must not be changed.
- `retn`: pseudo instruction
- `ret`: AArch64 instruction

Reply:
Right! This is a blatant bug from me. Fixed in [force-push](https://github.com/eclipse/omr/compare/247afa7987508aa12dc1695227443eff091179eb..b224aa3d87dc11b7aa4740c0b6b7523e7cd75d5b).